chia/wallet/key_val_store.py

```diff
@@ -18,9 +18,9 @@ async def create(cls, db_wrapper: DBWrapper2):
         self.db_wrapper = db_wrapper
         async with self.db_wrapper.writer_maybe_transaction() as conn:
             await conn.execute("CREATE TABLE IF NOT EXISTS key_val_store(key text PRIMARY KEY, value blob)")
-
-            await conn.execute("CREATE INDEX IF NOT EXISTS key_val_name on key_val_store(key)")
-
+            # Remove an old redundant index on the primary key
+            # See https://github.com/Chia-Network/chia-blockchain/issues/10276
+            await conn.execute("DROP INDEX IF EXISTS key_val_name")
```

Comment:
neurosis' ticket says the name of this index is already used for a different index. If that's the case, it's not safe to just `DROP` it. However, I can't find it used anywhere else. Have you looked for it in our history?

Reply:
I didn't. I checked every index in the wallet DB and it was only used here.
As you found out, it's not used anywhere else, plus I didn't find any drop for that index anywhere else.